README.md

```diff
@@ -369,6 +369,21 @@ will give a more in-depth description of the command and available options.
  - `autograder.cli.courses.assignments.submissions.fetch.course.attempts` -- Get all the most recent submissions (code and grading output) for an assignment.
  - `autograder.cli.courses.users.list` -- List all the users in a course.
 
+#### Proxy Submissions
+
+Instructors and TAs may need to accomodate a student's unique situations.
+The autograder server includes proxy submissions so course staff may assist students if necessary.
+Proxy submissions allow course staff to target a `proxy-user` and set a `proxy-time`.
+
+ - `proxy-user` -- The student that the course staff is submitting on behalf of.
+ - `proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
+   - If you want to set the time manually, time is measured by the number of milliseconds from the [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time).
```

Comment:
The flowery language here is causing inaccuracies.
It is not "measured by ...", the time **is** the number of milliseconds since the Unix Epoch.